buildchain/buildchain/targets/remote_image.py

```diff
@@ -90,17 +90,34 @@ def _show(task: types.Task) -> str:
         return utils.title_with_target1('IMG PULL', task)
 
     def _build_actions(self) -> List[types.Action]:
+        """Compute actions for the image — pull, tag, save."""
         filepath = self.uncompressed_filename
+        pull_callable = DockerPull(
```

Comment:
I would rename `pull_callable` to `docker_pull` and `tag_callable` to `docker_tag`.